jbpm-case-mgmt/jbpm-case-mgmt-api/src/main/java/org/jbpm/casemgmt/api/model/CaseStatus.java

```diff
@@ -0,0 +1,25 @@
+package org.jbpm.casemgmt.api.model;
+
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+public enum CaseStatus {
+    PENDING,
+    OPEN,
+    CLOSED,
+    CANCELLED,
+    SUSPENDED
```

Comment:
I would remove PENDING and SUSPENDED as they do not make much sense for cases at the moment - I mean we don't have support for it